packages/attestation-service/config/.env.development

```diff
@@ -1,18 +1,26 @@
 DATABASE_URL=sqlite://db/dev.db
 CELO_PROVIDER=http://localhost:8545
-CELO_VALIDATOR_ADDRESS=0xE6e53b5fc2e18F51781f14a3ce5E7FD468247a15
+CELO_VALIDATOR_ADDRESS=x
 ATTESTATION_SIGNER_ADDRESS=x
 APP_SIGNATURE=x
-SMS_PROVIDERS=twilio
+SMS_PROVIDERS=nexmo,twilio
+SMS_PROVIDERS_US=twilio,nexmo
+SMS_PROVIDERS_MX=nexmo
+EXTERNAL_CALLBACK_HOSTPORT=http://host:3000
+PORT=3000
 NEXMO_KEY=x
 NEXMO_SECRET=x
-NEXMO_BLACKLIST=
+NEXMO_UNSUPPORTED_REGIONS=
+NEXMO_ACCOUNT_BALANCE_METRIC=0
 TWILIO_ACCOUNT_SID=x
 TWILIO_MESSAGING_SERVICE_SID=x
 TWILIO_AUTH_TOKEN=x
 TWILIO_ACCOUNT_SID=x
```

Comment:
duplicate to 3 lines up

Reply:
not sure what you mean?

Reply:
oh this didn't post just the single line...and i didn't provide good explanation...

we have two `TWILIO_ACCOUNT_SID` parameters... its an old issues,  but I noted when trying to run and thought it could be remove with this pr

Reply:
oh yeah -- I'll remove one!